fms/models/hf/gpt_bigcode/configuration_gpt_bigcode_hf.py

```diff
@@ -0,0 +1,82 @@
+from typing import Optional
+
+from transformers import PretrainedConfig
+
+from fms.models.gpt_bigcode import GPTBigCodeConfig
+
+
+class GPTBigCodeHFConfig(PretrainedConfig):
+    model_type = "gpt_bigcode_hf"
+
+    attribute_map = {
+        "vocab_size": "src_vocab_size",
+        "hidden_size": "emb_dim",
+        "num_attention_heads": "nheads",
+        "num_hidden_layers": "nlayers",
+    }
+
+    def __init__(
+        self,
+        src_vocab_size: Optional[int] = 49280,
+        emb_dim: Optional[int] = 2048,
+        emb_kq: Optional[int] = None,
+        emb_v: Optional[int] = None,
+        nheads: int = 12,
+        kvheads: int = 0,
```

Comment:
Is GPTBigCode sometimes changing its kv-heads config?

Reply:
fixed this to only have multiquery_attn